### ISSUE TITLE
No option to disallow deleting characters.

### ISSUE THREAD

**Issue by tayrp** [BubbleDK/bub-multichar#3]
As said in the title, There is no option or config to get rid of the delete character buttons, I have a very specific way of going about character deletion.

 I add the line that disables it in default qbx_core to the multi-character config and still does not disable deletion of characters.

**Comment by noswear922** [comment]
Yes i love to havbe this option and i like how i dont have the option to select last location is perfect just need delete buton option
